.circleci/config.yml

```diff
@@ -147,7 +147,7 @@ commands:
               export RELEASE_TAG='-t release'
             fi
             mkdir -p logs
-            make html O="-T $RELEASE_TAG -j4 -w /tmp/sphinxerrorswarnings.log"
+            make html O="-T $RELEASE_TAG -j1 -w /tmp/sphinxerrorswarnings.log"
```

Comment:
This seems un-related?

Reply:
Woops, this definitely got applied to the wrong branch 🙈